site/docs/api-quickstart.md

```diff
@@ -63,6 +64,7 @@ val table = tables.create(schema, spec, "hdfs:/tables/logging/logs")
 // write into the new logs table with Spark 2.4
 logsDF.write
     .format("iceberg")
+    .mode("overwrite")
```

Comment:
Why use overwrite instead of append?

Reply:
Without specifying anything:
```
scala> val df = spark.read.format("csv").option("header", "true").load("/Users/fokkodriesprong/Desktop/presentations/delta-and-others/data/beers.csv")
df: org.apache.spark.sql.DataFrame = [_c0: string, abv: string ... 6 more fields]

scala> df.write.format("iceberg").save("/tmp/fokko/")
java.lang.IllegalArgumentException: Save mode ErrorIfExists is not supported
  at org.apache.iceberg.shaded.com.google.common.base.Preconditions.checkArgument(Preconditions.java:217)
  at org.apache.iceberg.spark.source.IcebergSource.createWriter(IcebergSource.java:89)
  at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:255)
  at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:229)
  ... 49 elided
```
Append does not work (because there isn't anything to append to:
```
scala> df.write.format("iceberg").mode("append").save("/tmp/fokko/")
org.apache.iceberg.exceptions.NoSuchTableException: Table does not exist at location: /tmp/fokko/
  at org.apache.iceberg.hadoop.HadoopTables.load(HadoopTables.java:83)
  at org.apache.iceberg.spark.source.IcebergSource.findTable(IcebergSource.java:125)
  at org.apache.iceberg.spark.source.IcebergSource.getTableAndResolveHadoopConfiguration(IcebergSource.java:151)
  at org.apache.iceberg.spark.source.IcebergSource.createWriter(IcebergSource.java:92)
  at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:255)
  at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:229)
  ... 49 elided
```

Reply:
You shouldn't see NoSuchTableException because the table is created above:

```java
val table = tables.create(schema, spec, "hdfs:/tables/logging/logs")
```

Reply:
Yes, something went wrong on my side. Append works fine.